src/content/language/constructors.md

```diff
@@ -211,20 +196,34 @@ class Vector3d extends Vector2d {
 }
 ```
 
-Super-initializer parameters cannot be positional 
-if the super-constructor invocation already has positional arguments,
-but they can always be named:
+To further illustrate, consider the following example.
 
-<?code-excerpt "misc/lib/language_tour/classes/super_initializer_parameters.dart (named)" plaster="none"?>
+```dart
+  // If you invoke the super constructor (`super(0)`) with any
+  // positional arguments, using a super parameter (`super.x`)
+  // returns an error.
```

Comment:
[nit] I don't know what "returns" would mean in this context. How about "is an error" or "results in an error"?

```suggestion
  // is an error.
```